src/Documentation/Getting-Started-With-Orleans/Developing-a-Client.md

```diff
@@ -3,71 +3,81 @@ layout: page
 title: Developing a Client
 ---
 
+### What Is Grain Client?
 
-Once we have our grain type implemented, we can write a client application that uses the type.
+The term "Client" or sometimes "Grain Client" is used for application code that interacts with grains but itself is not part of a grain logic.
+Client code runs outside of the cluster of [Orleans servers called "silos"](Silos.md) where grains are hosted.
+Hence, a client acts as a connector or conduit to the cluster and to all grains of the application.
 
-The following Orleans DLLs from either the `[SDK-ROOT]\Binaries\PresenceClient_ or _[SDK-ROOT]\Samples\References` directories need to be referenced in the client application project:
+![](Frontend-Cluster.png)
 
-* Orleans.dll
-* OrleansRuntimeInterfaces.dll
+Usually, clients are used on the frontend web servers to connect to an Orleans cluster that serves as a middle tier with grains executing business logic.
+In a typical setup, a frontend web server:
+* Receives a request over HTTP (or some other protocol)
+* Performs necessary authentication and authorization validation
+* Decides which grain(s) should process the request
```

Comment:
Maybe "Route the request to the controller which has the logic handling it" is more accurate 

"Decides" is not clear to me about how the server decide.